### ISSUE TITLE
Overlapping elements on a particular webpage

### ISSUE THREAD

**Issue by prosoitos** [qutebrowser/qutebrowser#6714]
**Version info**:

utebrowser v2.3.1
Git commit: 
Backend: QtWebEngine 5.15.6, based on Chromium 87.0.4280.144
Qt: 5.15.2

CPython: 3.9.7
PyQt: 5.15.4

**Does the bug happen if you start with `--temp-basedir`?**:

Yes

**Description**

Elements are overlapping on this page: https://www.westgrid.ca//events/westgrid_announces_webinars_and_training_workshops_fall_2021_0

It is the first time I notice this, so very few websites might be affected.

**How to reproduce**

Open this page: https://www.westgrid.ca//events/westgrid_announces_webinars_and_training_workshops_fall_2021_0

**Comment by toofar** [comment]
The fixed width of the table is greater than it's parent width. The relevant parent element is set to 66% percent of the div.container element which has max-width. So unless there is some media queries in the CSS we aren't triggering I assume it isn't qutebrowser specific.

**Comment by The-Compiler** [comment]
Strangely enough, it looks correct in Chromium:

![image](https://user-images.githubusercontent.com/625793/135035386-b7cc506b-4547-46d9-8e3a-7240c498fb1c.png)

in qutebrowser:

![image](https://user-images.githubusercontent.com/625793/135035423-adb1443c-3664-4625-864f-af8924d2d897.png)

**Comment by The-Compiler** [comment]
I can see the same issue in QtWebEngine's Simple Browser example, both with Qt 5.15 and 6.2.

I *don't* see the issue with an up-to-date Chromium, but I do see it with older Chromiums. Thus I suppose it's an issue which has been fixed somewhere after Chromium 90.0.4430.228 (coming with Qt WebEngine 6.2).

After trying to [bisect](https://www.chromium.org/developers/bisect-builds-py) it a bit in Chromium I unfortunately didn't find out where it was fixed, as for some reason most builds don't actually start on my machine... I modified their script a bit to ignore crashing builds and will run it in the background a bit, will reply again if I find out more.

Closing this for now as it seems pretty clear this is/was a Chromium issue.
